minimal/Content/package.json

```diff
@@ -1,14 +1,16 @@
 {
   "private": true,
   "scripts": {
-      "start": "webpack-dev-server"
+    "start": "npm-run-all --parallel fable:watch dev:watch",
```

Comment:
We usually use the command `dotnet fable watch src --run webpack-dev-server` to run Webpack after first watch compilation in one go. That way we wouldn't need the npm-run-all dependency, what do you think?

Reply:
Sure, I wasn't aware I could do that, node reflexes took over.

I change it and rebased.